pkg/gofr/gofr.go

```diff
@@ -309,87 +311,102 @@ func (a *App) initTracer() {
 
 	traceExporter := a.Config.Get("TRACE_EXPORTER")
 	tracerURL := a.Config.Get("TRACER_URL")
+	authHeader := a.Config.Get("TRACER_AUTH_KEY")
 
-	// deprecated : tracer_host and tracer_port is deprecated and will be removed in upcoming versions
+	// deprecated : tracer_host and tracer_port are deprecated and will be removed in upcoming versions.
 	tracerHost := a.Config.Get("TRACER_HOST")
-	tracerPort := a.Config.GetOrDefault("TRACER_PORT", "9411")
+	tracerPort := a.Config.Get("TRACER_PORT")
```

Comment:
why have we removed the default value?

Reply:
These env variables are no longer used. 

The idea is to report them as deprecated.

The default value implies we don't know if the port env variable was present or not.

Removing it allow to detect in tracer_host or tracer_port are present and ask user to remove them

Reply:
But wouldn't that be a breaking change?

Reply:
We are already logging a warning in case user is using host and port instead of url..
I believe that is reporting them

Reply:
the default value is for  Zipkin's port , we add otel collecter and  jaeger . the latter using 4317 as the default port , so i think remove it should be better .

Reply:
this would still be a breaking change. right?

Reply:
yeah ,  if some one using zipkin but not set  with a port

Reply:
correct, and I believe we should not break something for someone using GoFr.
So, can we pls revert?

Reply:
either remove or add i have no problem